app/acknowledgement/process-invalid.js

```diff
@@ -1,27 +1,26 @@
 const db = require('../data')
-const { getHoldCategoryId } = require('../holds')
-const getHoldCategoryName = require('./get-hold-category-name')
-const holdAndReschedule = require('../reschedule')
 const { resetPaymentRequestById } = require('../reset')
+const { getHoldCategoryName } = require('./get-hold-category-name')
+const { getHoldCategoryId } = require('../holds')
+const { holdAndReschedule } = require('../reschedule')
 const { sendAcknowledgementErrorEvent } = require('../event')
 
-const config = require('../config')
-
 const processInvalid = async (schemeId, paymentRequestId, frn, acknowledgement) => {
   const transaction = await db.sequelize.transaction()
   try {
-    await resetPaymentRequestById(paymentRequestId, schemeId, transaction)
+    await resetPaymentRequestById(paymentRequestId, transaction)
```

Comment:
It never should have been.  The `schedule` table doesn't have a `schemeId` column.  So all these functions pass it down just so it can be added but it's ignored.